src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -313,15 +376,30 @@ public void LoadBytes()
         [Fact]
         public void ReflectionOnlyGetAssemblies()
         {
-            Assert.Equal(Array.Empty<Assembly>(), AppDomain.CurrentDomain.ReflectionOnlyGetAssemblies());
+            RemoteInvoke(() => {
+                Assert.Equal(0, AppDomain.CurrentDomain.ReflectionOnlyGetAssemblies().Length);
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void MonitoringIsEnabled()
         {
-            Assert.False(AppDomain.MonitoringIsEnabled);
-            AssertExtensions.Throws<ArgumentException>(null, () => {AppDomain.MonitoringIsEnabled = false;});
-            Assert.Throws<PlatformNotSupportedException>(() => {AppDomain.MonitoringIsEnabled = true;});
+            RemoteInvoke(() => {
```

Comment:
If this works because of the bug, this will affect other test cases. Currently build time is considerably larger than all tests wrapped in RemoteInvoke. I wouldn't fight for few microseconds.